vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/storage/SessionStorageManager.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.ui.web.action.storage;
+
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import org.openqa.selenium.html5.SessionStorage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SessionStorageManager implements IBrowserStorageManager
```

Comment:
copy-paste should be eliminated, it should be enough to have a single class will accept the type of the storage and its supplier as input constructor parameters